raphtory-graphql/src/lib.rs

```diff
@@ -170,7 +153,9 @@ mod graphql_test {
 
         let graphs = HashMap::from([("graph".to_string(), graph)]);
         let tmp_dir = tempdir().unwrap();
-        let data = Data::new(tmp_dir.path(), Some(graphs), None, &AppConfig::default());
```

Comment:
Don't get rid of the `default` implementation for `AppConfig`, it is a nice API, just implement it using `AppConfigBuilder::new().build()`.